From 162dcef92807a93eaa0721c5a487e9bd57b75045 Mon Sep 17 00:00:00 2001
From: Franck Bui <fbui@suse.com>
Date: Thu, 24 Mar 2016 21:36:08 +0100
Subject: [PATCH 1001/1003] journald: turn ForwardToSyslog= on by default

On SLE products, rsyslog is used as main daemon logger and journald
uses to forwarding messages to it.

This basically revert 46b131574fdd7d77c15a0919ca9010cad7aa6ac7.

[fbui: fixes bsc#1065301]
---
 man/journald.conf.xml         | 2 +-
 src/journal/journald-server.c | 1 +
 src/journal/journald.conf     | 2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/man/journald.conf.xml b/man/journald.conf.xml
index bfd359a903..288ccc8064 100644
--- a/man/journald.conf.xml
+++ b/man/journald.conf.xml
@@ -338,7 +338,7 @@
         traditional syslog daemon, to the kernel log buffer (kmsg), to the system console, or sent as wall
         messages to all logged-in users.  These options take boolean arguments. If forwarding to syslog is
         enabled but nothing reads messages from the socket, forwarding to syslog has no effect. By default,
-        only forwarding to wall is enabled. These settings may be overridden at boot time with the kernel
+        only forwarding syslog and to wall are enabled. These settings may be overridden at boot time with the kernel
         command line options <literal>systemd.journald.forward_to_syslog</literal>,
         <literal>systemd.journald.forward_to_kmsg</literal>,
         <literal>systemd.journald.forward_to_console</literal>, and
diff --git a/src/journal/journald-server.c b/src/journal/journald-server.c
index 5865bf9809..c89baec34b 100644
--- a/src/journal/journald-server.c
+++ b/src/journal/journald-server.c
@@ -2218,6 +2218,7 @@ int server_init(Server *s, const char *namespace) {
                 .ratelimit_interval = DEFAULT_RATE_LIMIT_INTERVAL,
                 .ratelimit_burst = DEFAULT_RATE_LIMIT_BURST,
 
+                .forward_to_syslog = true,
                 .forward_to_wall = true,
 
                 .max_file_usec = DEFAULT_MAX_FILE_USEC,
diff --git a/src/journal/journald.conf b/src/journal/journald.conf
index 2e1aacd8c5..d6cd5b1519 100644
--- a/src/journal/journald.conf
+++ b/src/journal/journald.conf
@@ -29,7 +29,7 @@
 #RuntimeMaxFiles=100
 #MaxRetentionSec=
 #MaxFileSec=1month
-#ForwardToSyslog=no
+#ForwardToSyslog=yes
 #ForwardToKMsg=no
 #ForwardToConsole=no
 #ForwardToWall=yes
-- 
2.26.2

